platform/kubernetes/kubelet/client.go

```diff
@@ -10,7 +10,8 @@ import (
 	"path"
 	"regexp"
 
-	kubeletTypes "github.com/mackerelio/mackerel-container-agent/internal/k8s-apis/stats/v1alpha1"
+	kubeletTypes "k8s.io/kubelet/pkg/apis/stats/v1alpha1"
+
```

Comment:
[nit] Maybe this and the next file's empty lines are redundant?

Reply:
fixed at 1a865cd54b3a4eaf808af297924058c82843066a, f1fcafdc5e2112b1334be63ba083dfc0b2dc691f